LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -53,10 +53,14 @@
                 <div id="groupList" class="leaf-displayFlexRow"></div>
             </div>
         </div>
-
     </main>
-
-
+</div>
+<!--Loading Modal-->
+<div class="loadingModal">
```

Comment:
Since we are likely to use this modal in multiple places, I would convert it to it's own Smarty template and use the variable passing of {include} Smarty function to call the template file with say just the "User Group" as a variable.  In this way, we can modularize the modal to make it easier to add (one layout, multiple values) https://www.smarty.net/docsv2/en/language.function.include.tpl

Reply:
This is exactly what I was looking for. Awesome.